option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message VisitFriendAccelerateRequest
{
 required U64Type player_id = 1;
 repeated uint32 route_id = 2;
}
